Makefile

```diff
@@ -138,8 +138,13 @@ boolector: smt
 btormc: mc
 	$(foreach file, $(btor2s), btormc $(file);)
 
+# Test testimony for all C* files in sysmbolic
+testimony: tools/Testimony.py
+	chmod 755 tools/Testimony.py
+	$(foreach file, $(wildcard symbolic/*.c), ./tools/Testimony.py $(file);)
+
 # Run everything
-everything: all assemble spike qemu boolector btormc grader grade
+everything: all assemble spike qemu boolector btormc grader grade testimony
```

Comment:
You need a better name for your tool. How about `validator`?

Reply:
Done.